WordPress/gutenberg
# Font Library: fix to activate and display the right activation state of system fonts (fonts with no font faces)

## What?
 fix to activate and display the right activation state of system fonts (fonts with no font faces)
## Why?
Because system fonts were failing to be activated and display if they were active or not.

## How?
checking if there are font faces available for a font family.

## Testing Instructions
1. Add a collection that features system fonts (install and activate the collection that this plugin offers in this PR: https://github.com/matiasbenedetto/modern-fonts-stacks-for-wp-font-library/pull/1 )
2. Install a font family from that collection
3. Check that the font family can be installed as expected and the UI works correctly to display if its activated or not.
